src/RPackage-Core/UnpackagedPackage.class.st

```diff
@@ -0,0 +1,186 @@
+"
+I represent the package of unpackaged classes. (Null object pattern)
+
+I will return empty collection for everything we can ask me and I'll be returned if we ask the package of an unpackaged class.
+"
+Class {
+	#name : #UnpackagedPackage,
```

Comment:
Maybe UndefinedPackage would be a better name. I always found the current name a bit weird

Reply:
For now I wanted to keep this name to make the change easier since the name is used in multiple places in the system